index.d.ts

```diff
@@ -8,18 +8,20 @@ export interface StoreCreator {
   ): Store<S>;
 }
 
-export type Loop<S, A extends Action> = [S, CmdType<A>];
+type WithDefaultActionHandling<T> = T | Action<'ENFORCE_DEFAULT_HANDLING'>;
```

Comment:
can you throw on a prefix like @@REDUX_LOOP/ENFORCE_DEFAULT_HANDLING to avoid name conflicts

Reply:
also would it make sense to say `T extends AnyAction`?

Reply:
Yep, both good points.